Userland/Libraries/LibWeb/UIEvents/MouseEvent.cpp

```diff
@@ -96,6 +96,32 @@ bool MouseEvent::get_modifier_state(String const& key_arg) const
     return false;
 }
 
+void MouseEvent::init_mouse_event(String const& type, bool bubbles, bool cancelable, HTML::Window* view, WebIDL::Long detail, WebIDL::Long screen_x, WebIDL::Long screen_y, WebIDL::Long client_x, WebIDL::Long client_y, bool ctrl_key, bool alt_key, bool shift_key, bool meta_key, WebIDL::Short button, DOM::EventTarget* related_target)
```

Comment:
Can we add a spec link for this method?

Reply:
Absolutely, my mistake - I've amended this :^)